### PR TITLE
Termotto2

### PR DESCRIPTION
**Pull request by CustomDesignBuildStudios**
Requesting to add all these cool new features into the group app!

### CODE REVIEW

File: termotto/termotto.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+print("This is the first commit")
+
+print ("Feature One Added")
```
Comment (CustomDesignBuildStudios) [inline termotto/termotto.py:3]: Feature one is lacking in features

File: termotto2/termotto2.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+print("Main Branch File Added")
+
+print("Termotto2 branch added feature")
```
Comment (CustomDesignBuildStudios) [inline termotto2/termotto2.py:3]: Termotto2 feature is really nice.

**Review by CustomDesignBuildStudios — COMMENTED** [review]
I believe these features will increase user engagement.

**Comment by BDubOne** [comment]
Yeaaaah buddy

**Comment by seanmac-dev** [comment]
Excellent, Termotto2 is noooiiiiice!

**Pull request merged**